app/assets/stylesheets/application.scss

```diff
@@ -3,45 +3,42 @@ $govuk-compatibility-govukelements: true;
 $govuk-use-legacy-palette: false;
 $covid-grey: #272828;
 $covid-yellow: #fff500;
-
-
+$covid-hover: #b0c3e1;
```

Comment:
This may be better labelled as `$covid-action-link-hover`

Reply:
updated as suggested.